Infrastructure.DataAccess/Migrations/202411180725295_Add country code field to organization.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Infrastructure.DataAccess.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class Addcountrycodefieldtoorganization : DbMigration
```

Comment:
This migration is empty. I think the migrations need another cleanup and combining into 1